ali_ecc_golang.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"bytes"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math/rand"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/tendermint/tendermint/crypto/secp256k1"
+)
+
+var (
+	appId         = "25dzX3vbYqktVxyX"
+	deviceId      = "URJuHA0FgRACAW8eGUh/zJ+2"
```

Comment:
Where did you get the `deviceId` value from?

Maybe it's like `e5173011-XXXX-XXXX-XXXX-XXXXf04f3c62` or uuid string?

Reply:
I checked it in my browser, it seems to be random.

Reply:
It sounds emmm... I know